.storybook/static/preview-head.js

```diff
@@ -375,3 +375,44 @@ function toggleNestedListSubmenu(event) {
         icon.classList = 'sap-icon--navigation-down-arrow';
     }
 }
+
+function collectionHas(a, b) { //helper function (see below)
+    for(var i = 0, len = a.length; i < len; i ++) {
```

Comment:
Why not to use `let` instead of `var`?

also, I'd suggest putting empty space after `for` for better formatting